### DISCUSSION TITLE
Reverse screen

### DISCUSSION THREAD

**Discussion by 4ndy1** [egzumer/uv-k5-firmware-custom#424] [General]
How do I get the reverse screen on uv5r plus quansheng I’ve got egzumer firmware installed

**Comment by OE1MWW** [comment]
the reverse screen is not implemented in any of the egzumer versions

**Reply by 4ndy1** [reply]
Does the quansheng uv5r stay on am mode in the 225-400 MHz band as when I do a search it’s go am display on bottom of screen would be good for military airband if so

**Reply by rpcsinc** [reply]
I have the uvk5-uvk6 if you can get his firmware on yours then yes it'll scan the mil bands.

**Reply by rpcsinc** [reply]
Just have an antenna that will pick up in that spectrum. I have decent luck with an am loop or my radtel airband whip.

**Comment by 4ndy1** [comment]
Ok thank you

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by 4ndy1** [comment]
Is there a version I could use and still have good am mode and strength meter

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by 4ndy1** [comment]
Is the egzumer firmware good for 225-400mhz am as it's military band and would be perfect

**Comment by 4ndy1** [comment]
And they could possibly pick up military clear

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by rpcsinc** [reply]
Most comms will be encrypted, some won't. If it sounds like a machine gun or something it's encrypted. Also data transmissions like coordinates etc. Will come through, but without the right software it's just noise.  

You'd really like SDR. Check out RTL-SDR Blog v4 or V3. You can really listen into all the bands. There's decryption software out there. Some can be unencrypted some can't.

**Comment by 4ndy1** [comment]
I’m using egzumer.22 firmware

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________
